packages/bpk-component-inset-banner/src/BpkInsetBanner.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkInsetBanner.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  title?: string;
+  subheadline?: string;
+  logo?: string;
+  backgroundColor: string;
+  variant: 'onLight' | 'onDark';
```

Comment:
Can this be changed to have an exported object of `VARIANTS`

So something like:

```tsx
export const VARIANTS = {
  onLight: 'on-light',
  onDark: 'on-dark'
} as const;
```

Which will allow consumers to supply variant of `variant={VARIANT.onLight}`

Which can then mean this type can be 

```suggestion
  variant: (typeof VARIANT)[keyof typeof VARIANT];
```